src/helpers/pythonToFrames.ts

```diff
@@ -305,10 +306,43 @@ function transformCommentsAndBlanks(codeLines: string[], format: "py" | "spy") :
             }
             checkRearrangeCommentsIdent();
         }
+        else if(codeLines[i].trim().startsWith("'''") || codeLines[i].trim().endsWith("'''")){
+            // Note: we ignore the triple quotes *strings* in our code.
+            const indexOfFirstToken = codeLines[i].indexOf("'''");
+            if(codeLines[i].trim().startsWith("'''") && codeLines[i].trim().endsWith("'''") && codeLines[i].trimStart().length >= 6){
+                // If we have triple quote token both at the start and at the end, it's actually a single line comment...
```

Comment:
It can also be the end of one and start of next...  (I don't mind too much if we don't support this.)